src/SprykerEco/Zed/Payone/Business/Payment/Hook/PostSaveHook.php

```diff
@@ -139,7 +139,7 @@ protected function checkApiLogRedirectAndError(QuoteTransfer $quoteTransfer, Che
         $errorCode = $apiLogTransfer->getErrorCode();
 
         if ($errorCode) {
-            $checkoutResponse->addError($this->createCheckoutErrorTransfer($apiLogTransfer->getErrorMessageUser(), $errorCode));
+            $checkoutResponse->addError($this->createCheckoutErrorTransfer($apiLogTransfer->getErrorMessageUserOrFail(), $errorCode));
```

Comment:
```suggestion
            $checkoutResponse->addError($this->createCheckoutErrorTransfer((string)$apiLogTransfer->getErrorMessageUser(), $errorCode));
```

I'm not sure why we want to throw an exception here? This will interrupt the app for no obvious reason. Can't we cast this to string? What is the reason this is orFail()?

Reply:
The reason is only strict "string" param of `createCheckoutErrorTransfer()`.
Yes, it's possible to cast it.
Fixed.